apps_packaging.md

```diff
@@ -0,0 +1,14 @@
+# Apps packaging in progress
```

Comment:
Hmmm but those apps are expected to be listed in https://yunohost.org/#/apps, though you gotta choose "All apps" to actually see those ... But I'm really not sure we want to add an additional page like this one to maintain on the long term :/

Reply:
I partly agree with you, that's how I found the links. The proposal results from this [exchange](https://github.com/YunoHost/doc/pull/1056) with @frju365. It was a question for me, and in a longer-term perspective you're right. 

The ultimate solution is to delete these proposals as soon as they are submitted?

Reply:
I totally agree with your analysis! I don't realize what this implies at the technical level.